app/views/editions/_major_change_fields.html.erb

```diff
@@ -1,9 +1,15 @@
-<%= f.input :change_note,
-      as: :text,
-      label: 'Summary of changes to this edition',
-      input_html: { rows: 3, class: 'input-md-7', disabled: @resource.locked_for_edits? } %>
-<%= f.input :major_change,
-      as: :boolean,
-      input_html: {disabled: @resource.locked_for_edits?},
-      wrapper_html: {class: 'emphasised-field add-bottom-margin input-md-7'},
-      label: "Alert the public to this change" %>
+<div class="form-group" id="edition_change_note_input">
+  <span class="form-label">
+    <%= f.label :change_note, "Summary of changes to this edition" %>
+  </span>
+  <span class="form-wrapper">
+    <%= f.text_area :change_note, rows: 3, disabled: @resource.locked_for_edits?, class: "input-md-7 form-control" %>
+  </span>
+</div>
+
+<div class="form-group emphasised-field add-bottom-margin">
+  <span class="form-wrapper">
+    <%= f.check_box :major_change, disabled: @resource.locked_for_edits?, class: "input-md-7 form-control" %>
+    <%= f.label :major_change, "Alert the public to this change", class: "checkbox-label" %>
```

Comment:
Just letting you know that I think I did something like this https://github.com/alphagov/support/pull/820/files#diff-88a413ee564f782597dbd0d45ece710fR67-R74 to get the correct styling on these boolean checkboxes without having to add a css class. But I have no strong opinion on this so up to you